src/clusterfuzz/_internal/bot/tasks/setup.py

```diff
@@ -467,6 +467,12 @@ def _prepare_update_data_bundle(fuzzer, data_bundle):
   return data_bundle_directory
 
 
+def take_trusted_host_path():
+  if environment.is_uworker():
```

Comment:
Would it make sense to move this check into `is_trusted_host`? or is it specific to this case?